eureka/S3_data_reduction/plots_s3.py

```diff
@@ -206,6 +206,7 @@ def profile(eventdir, profile, submask, n, hide_plots=False):
     -------
     None
     '''
+    profile = np.ma.masked_invalid(profile)
     vmax = 0.05*np.max(profile*submask)
```

Comment:
You'll likely want to change this to np.ma.max

Reply:
Fixed, thanks